lib/scsi.h

```diff
@@ -874,6 +874,57 @@ struct UnmapBlockDescriptor {
 } ABSL_ATTRIBUTE_PACKED;
 static_assert(sizeof(UnmapBlockDescriptor) == 16);
 
+// SCSI Reference Manual Table 164
+// https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+struct RequestSenseCommand {
+  OpCode opcode : 8;
```

Comment:
Other SCSI commands don't have the opcode field. It would be good to maintain consistency